content/en/tracing/trace_collection/custom_instrumentation/python/ddtrace-api.md

```diff
@@ -0,0 +1,256 @@
+---
+title: Python Custom Instrumentation using ddtrace-api-py (pre-release)
+description: '(pre-release) Manually instrument your Python application to send custom traces to Datadog.'
+code_lang: dd-api-alpha
+type: multi-code-lang
+code_lang_weight: 1
+further_reading:
+    - link: 'tracing/other_telemetry/connect_logs_and_traces'
+      tag: 'Documentation'
+      text: 'Connect your Logs and Traces together'
+    - link: 'tracing/glossary/'
+      tag: 'Documentation'
+      text: 'Explore your services, resources, and traces'
+---
+
+*Note: the ddtrace-api Python package is in a pre-release state and may contain significant issues. If you'd rather use something more stable, use the [ddtrace API][9].*
+
+If you have not read the setup instructions for automatic instrumentation, start with the [Python Setup Instructions][6]
+
+If you aren't using supported library instrumentation (see [library compatibility][1]), you may want to manually instrument your code. You may also want to gain finer control
+over your application's instrumentation.
+
+## Additional Setup
+
+The stable public API for Datadog APM's custom Python instrumentation is implemented by the [ddtrace-api package][8]. This package doesn't implement any of the tracing functionality that creates and
+sends spans to Datadog; it *only* implements the API interface. This separation between interface in `ddtrace-api` and implementation in `ddtrace` allows users of cusom instrumentation to rely on
+an API that changes less frequently and more predictably, while allowing autoinstrumentation users to ignore semantic versioning entirely.
```

Comment:
```suggestion
an API that changes less frequently and more predictably, while allowing autoinstrumentation-only users to ignore API changes.
```

We (currently) use semver for purposes other than API management, such as deprecating support for things. Let's keep semver language out of this for now, so as to not lock us in to anything.